packages/frontend-2/lib/viewer/composables/savedViews/state.ts

```diff
@@ -86,12 +88,12 @@ export const useViewerSavedViewIntegration = () => {
     await update({ settings })
   })
 
-  // Apply saved view state on initial load
-  useOnViewerLoadComplete(async ({ isInitial }) => {
-    if (isInitial) {
-      await apply()
-    }
-  })
+  // // Apply saved view state on initial load
```

Comment:
we might not need to do this, but leaving commented out version incase we need to add it back